bin/nvcc_wrapper

```diff
@@ -338,6 +364,10 @@ do
   shift
 done
 
+if [ $host_only -eq 0 -a $bad_nvcc_std -eq 1 ]; then
+  echo "nvcc_wrapper cannot compile for device with unsupported standard $bad_std_flag"
+fi
```

Comment:
ah we now just error out?

Reply:
Yes. CMake tests will run in "host mode" which means there is no longer any scenario where we need to accept bad flags.